lms/djangoapps/student_account/views.py

```diff
@@ -39,9 +39,13 @@
 )
 from openedx.core.lib.edx_api_utils import get_edx_api_data
 from openedx.core.lib.time_zone_utils import TIME_ZONE_CHOICES
-from openedx.features.enterprise_support.api import enterprise_customer_for_request, get_enterprise_learner_data
+from openedx.features.enterprise_support.api import enterprise_customer_for_request, get_enterprise_customer_for_learner
 from student.cookies import set_experiments_is_enterprise_cookie
-from openedx.features.enterprise_support.utils import update_third_party_auth_context_for_enterprise
+from openedx.features.enterprise_support.utils import (
+    update_login_and_registration_context_for_enterprise,
+    update_third_party_auth_context_for_enterprise,
```

Comment:
Since third party auth context will be available when we call the method `update_login_and_registration_context_for_enterprise` so I think we can merge `update_third_party_auth_context_for_enterprise` with `update_login_and_registration_context_for_enterprise`.